### DISCUSSION TITLE
Difficulties to using GrdiPatchDataset

### DISCUSSION THREAD

**Discussion by FerseinChris** [Project-MONAI/MONAI#7791] [Q&A]
![mmexport1716347220980](https://github.com/Project-MONAI/MONAI/assets/68042893/f2d1e036-4dc2-42dc-9817-66a29abf143f)
I have attached a picture showing my codes, I wish to create a gridpatchdataset. I've done it following the documentation here https://docs.monai.io/en/stable/data.html#gridpatchdataset , after creating the loader, I;m unable to even check the shape of the patches. Help check if the code is not correct and propose a solution to correctly building a gridpatchdataset for custom dataset and how to access data in the loader for further use.

**Comment by KumoLiu** [comment]
Hi @FerseinChris, here are some examples which include the usage of `GridPatchDataset` also include how to check the data shape after dataloader.
https://github.com/Project-MONAI/tutorials/blob/93c470587d98a6a10843824c23f9edb5658d15c6/modules/nifti_read_example.ipynb#L107
https://github.com/Project-MONAI/tutorials/blob/93c470587d98a6a10843824c23f9edb5658d15c6/modules/2d_slices_from_3d_training.ipynb#L22

Hope it helps, thanks!
